config/settings.yml

```diff
@@ -432,6 +432,9 @@ defaults: &defaults
       secret: 'service-fabrik-blueprint-secret'
```

Comment:
maybe its better to have a new plan to have support for k8s and leave the current one's for cf as the plan name in k8s has to be in different format ?!

Reply:
There will not be any new plan. The same existing BOSH plans will be shown for K8S as well.